### ISSUE TITLE
Mixin parsing error

### ISSUE THREAD

**Issue by davetgreen** [sasstools/sass-lint#184]
I've just started using sass-lint via grunt-sass-lint and I'm getting a parsing error due to this mixin that I have:

````
/// Input placeholder styling
/// @param {List} $styles - Styles
/// @requires $kp-ph--prefixes
/// @group forms
@mixin kp-placeholder-styles($styles) {
  @each $pseudo in $kp-ph--prefixes {
    @at-root &#{$pseudo} {

      @each $key, $value in $styles {
        #{$key}: #{$value};
      }
    }

    @at-root &:focus#{$pseudo} {
      color: transparent;
    }
  }
}
````

Producing this error:
![screen shot 2015-09-18 at 17 40 30](https://cloud.githubusercontent.com/assets/3214928/9965187/6e9f2678-5e2c-11e5-884b-d1a92c3c050f.png)

I've quickly copied the mixin and variables it depends on into Sassmeister (using Libsass 3.2.5 option) and it's working perfectly. I know this code is producing the correct results as I'm using it in a number of projects.

I was wondering whether this could be related to #44 but felt it best to post a new issue in case.

**Comment by davetgreen** [comment]
After a little testing it seems that it doesn't like more than one interpolated string per line?

**Comment by bgriffith** [comment]
This sounds as if it's related to #44

**Comment by DanPurdy** [comment]
Hmm, this looks like another issue with our AST, gonzales-pe.. I'm afraid we may be a little restricted in what we can do to fix this.. Will double check later on though...

**Comment by Undistraction** [comment]
FWIW this seems to fail on any interpolated string, quoted or otherwise, for example:

```
@mixin example {
    $name: 'width';
    #{$name}: 100%;
}
```

**Comment by DanPurdy** [comment]
Yeah @Undistraction we are just waiting for the latest release of our AST to happen, it should fix the problems with interpolated strings. Looks like under a week now. Fingers crossed!

**Comment by Undistraction** [comment]
@DanPurdy That's great news. Thanks for the headsup.

**Comment by DanPurdy** [comment]
See here for more info #222

**Comment by davetgreen** [comment]
@DanPurdy That's excellent news! Looking forward to kicking grunt-scss-lint out of bed once it drops! :+1:

**Comment by aarr0n** [comment]
:+1:

**Comment by DanPurdy** [comment]
The original issue above is still causing a parse error as of the latest build of gonzales-pe (3.2.1) I've filed it as a fresh issue to hopefully see this fixed. You can follow the issue [here](https://github.com/tonyganch/gonzales-pe/issues/116)

**Comment by davetgreen** [comment]
@DanPurdy Thanks for the update, much appreciated

**Comment by davetgreen** [comment]
@DanPurdy now that the issue is fixed in gonzales-3.2.x, are we just waiting for a sass-lint update which includes that update?

**Comment by DanPurdy** [comment]
Yes, unfortunately gonzales-pe isn't installing on windows at moment due to some bash specific post install scripts so we're waiting for them to update or if we'll just fork it and roll our own version of it soon. Fingers crossed it wont be too long!

**Comment by DanPurdy** [comment]
@davetgreen This is fixed in sass-lint 1.6.0

**Comment by davetgreen** [comment]
@DanPurdy Thank you so much!
